### DISCUSSION TITLE
Is there way to highlight when a bird had been detected for the first time?

### DISCUSSION THREAD

**Discussion by wepperly** [mcguirepr89/BirdNET-Pi#1126] [Ideas]
I keep noticing that the unique bird species detection number increments, but I do not see an easy way to determine what that species was.  I have been looking at the species recording page and looking for one I do not Rember seeing.

It would also be neat to see a graph that shows when species were first detected by date for the year.  That way we can easily see when birds start migrating into the area, or maybe highlight any bird that has not been detected for a couple months.

Did I overlook this.  It seems like it should be a feature, but I cannot find it.

Lots of systems give badges.  It would be like collecting badges/stickers.  Sounds stupid, but I suspect that is what people will want.

This is an awesome piece of software. Love it.

**Comment by stevedee** [comment]
Have you taken a look at Species Stats? This may give you what you want.

But just to note that BirdNet only detects bird calls.

One of the most numerous species in our garden is the woodpigeon, which will clock up a very high count on BirdNet during the spring & summer months, but is completely mute during the winter. Birds generally only make a noise to draw attention to themselves (which is risky) when trying to attract a mate or protect territory.

The chart below already shows that our local Blue Tits are trying to find mates as they break away from their winter flocks. By March/April the number of calls will increase and the number of actual birds will decrease (because each will try to defend their own territories).

![BlueTit_speciesChart](https://github.com/mcguirepr89/BirdNET-Pi/assets/1870807/f8bf03c3-17bf-44c6-95ec-8d081e361c12)

**Reply by wepperly** [reply]
I had not noticed that chart at the bottom of the bird species page.  That shows species activity over the long term. Thanks.

I was most interested in just a way to see what is new for the day.  I only set up my station last week.  It is still new.  I get 1 or 2 new species identified per day, but I have to go figure out what they are.  The daily stats page I think shows the top ten and the bottom ten.  It would be nice to also show first time detections of each species that happened during the day.  Or better yet, today's detections, add another number that takes the user to the list of new detections for the station. Like can be done for the unique species today.

![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/6520362/f96eb601-3909-492a-8cd4-90d2ea66a9c0)
